generators/app/templates/src/app/components/Routes/styles.scss

```diff
@@ -0,0 +1,3 @@
+.container {
```

Comment:
I'd remove `styles.js` if this one will replace it

Reply:
I did, I removed styles.js